### PR TITLE
Project 1: Code Review

### PR DESCRIPTION
**Pull request by Paul-js-hub**
### Microverse Code Review

In this project I:

- Created a local and remote repos
- Copied code from the previous admissions trial project.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+.form-input {
+    display: block;
+    width: 100%;
+    height: 2.375rem;
+    color: #495057;
+    background-color: #fff;
+    border: 1px solid;
+    border-radius: 0.25rem;
+}
```
Comment (Paul-js-hub) [inline css/style.css:33]: Please,  add two spaces of indentation. Do not use the tab key.  Tab key makes your code unreadable

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
+.header{
+    text-align: center;
+}
```
Comment (Paul-js-hub) [inline css/style.css:3]: Please, there should be space between the header class and the code blocks i.e curly braces. The absence of spaces makes your code unclean.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+          >Which option best describes your current role?</label
+        >
+        <select name="current-role" id="current-rolw" class="form-input">
+          <option value="student">Student</option>
+          <option value="fulltimejob">Full Time Job</option>
+          <option value="fulltimelearner">Full Time Learner</option>
+          <option value="prefernottosay">Prefer not to say</option>
+          <option value="other">Other</option>
+        </select>
+      </div>
```
Comment (Paul-js-hub) [inline index.html:70]: Kindly, implement the correct spacing in your form. Good padding and margins make your website look nice and appealing to the users. 
 
test1

File: index.html [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+      <div class="form-container">
+        <label id="name" for="name">Email</label>
+        <input
+          type="text"
+          name="name"
+          id="input-name"
+          class="form-input"
+          placeholder="Enter your email"
+        />
+      </div>
```
Comment (Paul-js-hub) [inline index.html:48]: Please, use the correct type for the email input. The type should be "type = email" and not text as you have written. This helps in avoiding validation errors.

**Review by Paul-js-hub — COMMENTED** [review]
# Hi! Great job

However, there are some changes required ♻️ 
Please, read through my comments and perform the required changes.

Happy coding! 👏👏👏

File: css/style.css (hunk 1 above)
Comment (phelian23) [reply css/style.css:33]: - [ ] Nicely done 👍

File: css/style.css (hunk 2 above)
Comment (phelian23) [reply css/style.css:3]: - [ ] Nicely done 👍

File: index.html (hunk 3 above)
Comment (phelian23) [reply index.html:70]: - [ ] Nicely done 👍

File: index.html (hunk 4 above)
Comment (phelian23) [reply index.html:48]: - [ ] Nicely done 👍

**Review by phelian23 — APPROVED** [review]
Hi @Paul-js-hub, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**